airbyte-integrations/connectors/destination-teradata/src/test/resources/config.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
was this file supposed to be included? (looks like .gitignore above is trying to exclude this file). If you meant to include a sample config file please prefix the file name with `sample_` and don't use real working credentials in it.

Reply:
No, this file is not using in tests instead using secrets/config.json. Deleted the file.